src/mongo/db/pipeline/expression.cpp

```diff
@@ -2144,74 +2145,81 @@ const char* ExpressionLog10::getOpName() const {
 
 /* ------------------------ ExpressionNary ----------------------------- */
 
+/**
+ * \brief Optimize a general Nary expression.
```

Comment:
We don't use Doxygen keywords like brief or return anywhere else in the codebase, please remove.